### PR TITLE
buttons are styled nicer

### PR DESCRIPTION
**Pull request by rosalie0**
All items entry buttons are styled nicer now

### CODE REVIEW

**Review by samdobson28 — COMMENTED** [review]
Yay! Let's increase the padding!

File: front/src/components/items/AllItems.js [hunk 1]
```diff
@@ -3,6 +3,40 @@ import { collection, getDocs } from "firebase/firestore";
 import { db } from "../firebase/firebase-config";
 
 export default function AllItems({ items }) {
+  const messageButton = {
+    border: "none",
+    outline: "none",
+    color: "whitesmoke",
+    backgroundColor: "#609966",
+    fontFamily: "san-serif",
```
Comment (samdobson28) [inline front/src/components/items/AllItems.js]: Let's choose a more specific font

**Review by samdobson28 — CHANGES_REQUESTED** [review]
See comments for formatting

File: front/src/components/items/AllItems.js (hunk 1 above)
Comment (rosalie0) [reply front/src/components/items/AllItems.js]: Ok I'll look into it

**Review by samdobson28 — APPROVED** [review]
Awesome ty!

**Pull request merged**